tests/ForStatements/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -57,9 +78,23 @@ contract ForStatements {
             a++;
         }
 
-        for (; ; ) {
+        for (;;) {
```

Comment:
why this one has no space in between anymore?

Reply:
have a look at #187

Reply:
oh yes I remember, so what about the scenario in line 97 below?